docs/cloud-functions/cloud-logs.mdx

```diff
@@ -5,32 +5,77 @@ slug: /cloud-logs
+
+- `warn`: The **warn** method is used to pass **warning** type log messages. A warning may occur whenever a user has given a wrong or undesired input. The **warn** log can be used in any cloud function as follows:
+    ```js
+    logging.warn("This is a warning message!")
+    ```
+
+- `error`: The **error** method helps log critical errors that are being encountered by the cloud function while being executed. Error logs can be used for catching and correcting these failures. The **error** log can be used as:
+    ```js
+    logging.error("This is an error message!")
+
+<img src={logMessages} alt="Rowy Logging Messages" />
 
 While creating a Rowy project, cloud logs can come really handy to thoroughly examine changes during deployment. 
 
-To access the Rowy Cloud Logs, navigate to the menu on the top right corner of your project table. Locate and click the **Cloud Logs** button. 
+## Accessing Cloud Logs
 
-<img src={cloudLogs} alt="Cloud Logs" />
+There are two ways to access Rowy Cloud Logs:
 
-This fires up the **Cloud Logs Interface**, wherein you can access the logs categorized as follows:
+1. To access the Rowy Cloud Logs, **navigate to the menu on the top right corner** of your project table. Locate and click the **Cloud Logs** button. 
 
-### Build Logs
-Build logs are enhanced console outputs of builds. They are represented by a structured list of the events which took place during the respective build.
+    <img src={cloudLogs} alt="Cloud Logs" />
 
-### Audit Logs
-Audit logs are used to document activity within the software systems of your Rowy project.
+2. Alternatively, User can also see logs specific to a column by clicking on the *logs* icon in **Column Settings**.
+
+    <img src={cloudLogs2} alt="Cloud Logs" />
+
+
+## Cloud Logs Interface
+
+The Rowy Logs Explorer is a tool that allows you to view and analyze the logs generated by your Rowy project.
 
-### Function Logs
-Function logs allow us to monitor and analyze the whereabouts of functions used in a Rowy project.
+It provides multiple **filters** to help you narrow down the logs you wish to view.
+
+<img src={cloudLogs3} alt="Cloud Logs" />
+
+The **Cloud Logs Interface** categorizes the logs into:
+
+### Extension Logs
+Extension logs possess all the logs specific to all the extensions used in a Rowy table. To filter out the logs specific to a particular extension, select the extension from the dropdown.
 
 ### Webhook Logs
-Webhook logs include a record of every webhook atempt that our project makes. Additionally, they document the information on the success or failure of each webhook call.
+Webhook logs include a record of every webhook atempt that Rowy makes. Additionally, they document the information on the success or failure of each webhook call. To filter out the logs specific to a particular webhook, select the webhook from the dropdown.
+
+### Column Logs
+Column logs act as a filter and display the logs specific to a column, selected in the dropdown.
```

Comment:
Column logs support 4 different types of columns - Action, Derivative, Connector and Default Values. Maybe worth to mention here